lib/metadata_pot/metadata_pot.rb

```diff
@@ -0,0 +1,55 @@
+# -*- encoding: utf-8 -*-
+
+def metadata_pot_file_path
+  File.join(locale_path, GettextSetup.config['project_name'] + '_metadata.pot')
+end
+
+def generate_new_pot_metadata
+  metadata = load_metadata_information
+  open(metadata_pot_file_path, 'w') do |f|
+    f << <<-DOC
+#
+#, fuzzy
+# msgid ""
+# msgstr ""
+"Project-Id-Version: PACKAGE VERSION\\n"
+"Report-Msgid-Bugs-To: \\n"
+"POT-Creation-Date: #{DateTime.now} \\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\\n"
+"Language-Team: LANGUAGE <LL@li.org>\\n"
+"MIME-Version: 1.0\\n"
+"Content-Type: text/plain; charset=UTF-8\\n"
+"Content-Transfer-Encoding: 8bit\\n"
+"X-Generator: Translate Toolkit 2.0.0\\n"
+#
+      DOC
```

Comment:
That's fine. I'll merge now and do some refactoring as a separate commit. In all, this functionality looks fine to me; I just want to refactor so that it's easier to add tests, and that changes can be quicker in the future.